orienteer-camel/src/main/java/org/orienteer/camel/Module.java

```diff
@@ -45,6 +46,8 @@ public void onInitialize(OrienteerWebApplication app, ODatabaseDocument db) {
 		app.setMetaData(CamelWidget.INTEGRATION_SESSIONS_KEY, new ConcurrentHashMap<String,CamelContext>());
 		app.mountPages("org.orienteer.camel.web");
 		app.registerWidgets("org.orienteer.camel.widget");
+		MethodManager.get().addModule(Module.class);
+		MethodManager.get().reload();
```

Comment:
Can MethodManager understand that cache should be reloaded? Imho if it just add new methods - cache might be invalidated.